sklearn/utils/themes.py

```diff
@@ -0,0 +1,200 @@
+"""
+The :mod:`sklearn.utils.themes` module provides a
+CSS template class for creating dynamic stylesheets.
+It exposes utilities and themes for styling displayed diagrams.
+"""
+
+AUTO = """
```

Comment:
Can this be moved back to `sklearn/utils/_estimator_html_repr.py`? This would make the diff smaller and easier to review. In the end, only the color related lines will change.